ndatumuremyi/my-brand
# implement landing page

### What does this PR do?

This PR implements a landing page

### Description of Task to be completed?

implement landing page

### How should this be manually tested?

Clone the [repo](https://github.com/ndatumuremyi/my-brand)
Checkout branch `feat-implement-landing-page-1gDpqA76`
Open the index.html in your browser to check the new homepage

### What are the relevant pivotal tracker stories?

[#1gDpqA76](https://trello.com/c/1gDpqA76/7-landing-page)
